sentences/en/homeassistant_HassGetState.yaml

```
@@ -3,33 +3,51 @@ intents:
   HassGetState:
     data:
       - sentences:
-          - (do you know|tell me|<what_is>) [the [current] (state|value) of] <name> [in <area>]
-        response: one
-        excludes_context:
-          domain:
-            - scene
-            - script
-      - sentences:
-          - is [the] [state of] <name> {on_off_states:state} [in <area>]
-        response: one_yesno
-        excludes_context:
-          domain:
-            - cover
+          - (is|are)[ there] any {on_off_domains:domain} {on_off_states:state}
+          - (is|are)[ there] any {on_off_domains:domain} ({on_off_states:state};in <area>)
+          - (is|are)[ there] any <area> {on_off_domains:domain} {on_off_states:state}
```

```suggestion
          - <are_any> {on_off_domains:domain} {on_off_states:state} [<in_area_floor>]
          - <are_any> {on_off_domains:domain} <in_area_floor> {on_off_states:state}
          - <are_any> <area> {on_off_domains:domain} {on_off_states:state}
          - <is> <area> {on_off_domains:domain} on
```
Use the new `<are_any> and <is>` expansions as well as `<in_area_floor>`. Additionally, I think it's easier to parse the optional area suffix on the first sentence than the `(foo;bar)` syntax, so that's split among the two lines.

Final suggestion is to add an `is area domain on?` here and leave `is area domain off?` in the `all` sentences.

Not sure if there is a way for us to only reference `on` from `on_off_states:state`. It may be useful to do that because we could add `[(switched|turned|set [to])] on` into the common `on_off_states`.